src/Eval.hs

```diff
@@ -89,30 +89,33 @@ eval ctx xobj@(XObj o info ty) preference resolver =
         unwrapLookup =
           fromMaybe
             (throwErr (SymbolNotFound spath) ctx info)
-        -- | Try all lookups performs lookups for symbols based on a given
-        -- lookup preference.
+
```

Comment:
Where did this comment go?

Reply:
It might be because it’s a docstring comment with `|`?

Reply:
Huh, weird. Will add it back – thanks for noticing!